cookbooks/prescribed_velocity/prescribed_velocity.cc

```diff
@@ -110,6 +110,51 @@ namespace aspect
   }
 
   /**
+   * This function retrieves the unit support points (in the unit cell) for the current element
+   * The DGP elements used when 'set Use locally conservative discretization = true' is enabled do
+   * not have support points. If these elements are in use, a ficticious support point at the cell
+   * center is returned. This is necessary for the internal velocities to be constrained as desired.
```

Comment:
-> is returned for each shape function that corresponds to the pressure variable, whereas the support points for the velocity are correct; the fictitious points don't matter because we only use this function when interpolating the velocity variable, and ignore the evaluation at the pressure support points.